week-8/Homework/mandatory/4-programmer-humour/app.js

```diff
@@ -0,0 +1,19 @@
+
+
+async function fetchData(){
+  const res = await fetch('https://xkcd.now.sh/?comic=latest')
+  const data = await res.json()
+    displayImg(data.img)
+}
+try {
+  fetchData();
+} catch (e) {
+  console.error(e);
+ } finally {
```

Comment:
A `finally` block isn't needed here since it doesn't do anything - you could remove it.